blingfire/src/lib.rs

```diff
@@ -22,13 +25,6 @@ pub fn text_to_sentences(source: &str, destination: &mut String) -> Result<()> {
 
 type Tokenizer = unsafe extern "C" fn(*const c_char, c_int, *mut c_char, c_int) -> c_int;
 
-// The maximum length is when the destination length can't fit in the maximum blingfire array. Worst
-// case, the source is tokenized by adding a space after every character + a null character at the
-// end.
-//
-// The maximum array size is defined in blingfire-sys/BlingFire/blingfireclient.library/inc/FALimits.h
-const MAX_SOURCE_LENGTH: usize = 1_000_000_000 / 2 - 2;
```

Comment:
I would keep this as `MAX_TEXT_LENGTH / 2 - 2`. But you can merge it either way, up to you.

Reply:
I still don't get why and it seems to rely on specifics of how the tokenizer works